package _22_exception;

import java.util.Random;
import java.util.Scanner;

public class ExceptionTest5 {
	public static void main(String[] args) {
		
//		int n = (int)(Math.random() * 99) + 1;
		Random random = new Random();
		int n = random.nextInt(99) + 1;
		
		
		try (Scanner sc = new Scanner(System.in)){
			
			while(true) {
				System.out.print("1 ~ 99 사이 정수를 입력하세요 >> ");
				if (!sc.hasNext()) {
					System.out.println("값이 잘못 입력 되었습니다.");
					continue;
				}
			}
			
		} 
		
	}
}
